_posts/2020-01-20-workshop.md

```diff
@@ -22,16 +22,16 @@ clinicians from around the world and discuss together and establish new collabor
+| 17:00 - 19:00 | <b>Session 2 - C3.09 Lecture Theatre (3rd floor)</b><br>John Ashburner (Institute of Neurology - UCL - UK)<br><i>Extending SPM for analysing images of head and neck</i><br><br>Ruud de Boer (Philips NL)<br><i>Recent developments in accelerating and optimizing spine and spinal cord sequences</i><br><br>Joe Yazhuo Kong (Wellcome Centre for Integrative Neuroimaging (WIN), University of Oxford - UK)<br><i>Spinal cord fMRI: what we can do so far?</i><br><br>Jon Stutters (Queen Square Multiple Sclerosis Centre - UCL - UK)<br><i>Data management lessons from a spinal cord study</i><br><br> |
 | 19:30 | <b>Social event / Dinner</b> |
 
 | 2nd Day | Tuesday, 21st January 2020<br>[Bentham House 124 - Gideon Schreier Lecture Theatre](https://www.ucl.ac.uk/laws/about/how-find-us)  |
 |---|---|
 | 09:00 - 11:00 | <b>Session 3</b><br>Claudia Wheeler-Kingshott (Queen Square Multiple Sclerosis Centre - UCL - UK)<br><i>Spinal cord as part of the central nervous system: advances in brain and spine acquisition</i><br><br>Gabriele de Luca (Oxford University, UK)<br><i>The pathological architecture of the Multiple Sclerosis spinal cord</i><br><br>Paola Valsasina (Milan, Italy)<br><i>Quantitative MRI of the spinal cord in multiple sclerosis</i><br><br>Rosanna Cortese (Queen Square Multiple Sclerosis Centre - UCL - UK)<br><i>Spinal cord involvement in myelin oligodendrocyte glycoprotein associated disease</i><br><br> |
 | 11:00 - 11:30 | Coffee break |
-| 11:30 - 13:30 | <b>Session 4</b><br>Olivier Vincent (Polytechnique Montreal - Canada)<br><i>Generalizing deep learning segmentation across multiple MRI contrasts and centers</i><br><br>Junqian Gordon Xu (Baylor College of Medicine - Houston - USA)<br><i>TBC</i><br><br>Govind Bhagavatheeshwaran (National Institute of Neurological Disorder and Stroke - USA)<br><i>Assessing the role of spinal cord atrophy in neuroinflammatory diseases</i><br><br>Frederik Barkhof (Centre for Medical Image Computing - UCL - UK)<br><i>Spinal cord atrophy rates in randomised controled trials</i><br><br> |
+| 11:30 - 13:30 | <b>Session 4</b><br>Olivier Vincent (Polytechnique Montreal - Canada)<br><i>Generalizing deep learning segmentation across multiple MRI contrasts and centers</i><br><br>Junqian Gordon Xu (Baylor College of Medicine - Houston - USA)<br><i>A pilot two-center reproducibility study of quantitative cervical spinal cord MRI for clinical trials in progressive multiple sclerosis</i><br><br>Govind Bhagavatheeshwaran (National Institute of Neurological Disorder and Stroke - USA)<br><i>Assessing the role of spinal cord atrophy in neuroinflammatory diseases</i><br><br>Frederik Barkhof (Centre for Medical Image Computing - UCL - UK)<br><i>Spinal cord atrophy rates in randomised controled trials</i><br><br> |
```

Comment:
He usually goes by the name "Govind Nair"

Reply:
ok